yew/src/lib.rs

```diff
@@ -136,8 +136,8 @@ pub mod events {
         } else if #[cfg(feature = "web_sys")] {
             #[doc(no_inline)]
             pub use web_sys::{
-                DragEvent, Event, FocusEvent, KeyboardEvent, MouseEvent, PointerEvent, TouchEvent, UiEvent,
-                WheelEvent,
+                self, DragEvent, Event, FocusEvent, KeyboardEvent, MouseEvent, PointerEvent,
```

Comment:
This will re-export `web_sys` in the `events` module. I think you meant to re-export at the lib level right?

Reply:
Oops, yes indeed.